src/components/MarketplaceItemDetailPage.vue

```diff
@@ -0,0 +1,174 @@
+          <img v-bind:src="item.thumbnailUrl" />
+        </div>
+        <div class="categories-container">
+          <h3>Category</h3>
+          <p>{{ item.category }}</p>
+        </div>
+        <div class="actions-container">
+          <button
+            v-on:click="goToProject(item)"
+            v-bind:data-tracking-label="item.name"
+            v-bind:href="item.sourceUrl"
+            class="btn btn--cta item-action-button"
+          >Take me to the Project</button>
+        </div>
+      </div>
+      <div class="right-panel">
+        <div class="description-container">
+          <h3>Description</h3>
+          <p>{{ item.description }}</p>
+        </div>
+        <div class="kentico-versions-container">
+          <h3>Works with Kentico versions</h3>
+          <span
+            v-for="version in item.kenticoVersions"
+            v-bind:key="version"
+            class="version-caption"
+          >{{ version }}</span>
+        </div>
+        <div class="version-container">
+          <h3>Version</h3>
+          <p>{{ item.version }}</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+Component.registerHooks(["beforeRouteEnter"]);
+
+import MarketplaceItemModel from "../models/MarketplaceItemModel";
+import store, { initializeStoreWithItemsAndNavigateNext } from "./../store";
+import { getItem } from "../utils/pathSegmentUtils";
+import { MARKETPLACE_ROOT_PATH_SEGMENT } from "./../constants/routes";
+import { TrackingEventType } from "../models/TrackingEventType";
+import { trackItemEvent } from "../utils/analyticsUtils";
+
+@Component
+export default class MarketplaceItemDetailPage extends Vue {
+  beforeRouteEnter(to: any, from: any, next: any) {
+    initializeStoreWithItemsAndNavigateNext(next);
+  }
+  beforeMount() {
+    const pathSegmentItemName = this.$route.params.itemName;
+    const allItems = this.$store.getters.allItems as MarketplaceItemModel[];
+
+    const renderedItem = this.item!;
+    document.title = renderedItem.name;
+    var metaElement = document.createElement("meta");
+    metaElement.setAttribute("name", "description");
+    metaElement.setAttribute("content", renderedItem.description);
+    document.getElementsByTagName("head")[0].appendChild(metaElement);
+
+    // cannot find item related to path segment
+    if (getItem(pathSegmentItemName, allItems) === null) {
+      this.$router.push(MARKETPLACE_ROOT_PATH_SEGMENT);
+    }
+  }
+
+  get item(): MarketplaceItemModel | null {
+    const pathSegmentItemName = this.$route.params.itemName;
+    const allItems = this.$store.getters.allItems as MarketplaceItemModel[];
+    return getItem(pathSegmentItemName, allItems);
+  }
+
+  goToProject(item: MarketplaceItemModel): void {
+    trackItemEvent(TrackingEventType.NavigateToExtensionProject, item);
+    window.open(item.sourceUrl, "_blank");
+  }
+}
+</script>
+
+<style scoped lang="scss">
```

Comment:
It is possible to wrap rule sets inscss.

```scss
.container {
   .left-panel {
   }
}
```

Reply:
I know, however, I don't need higher specificity and IMO ithis is easier for readibility.